frontend/.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
I am not familiar with this file, but it looks like it tries to read .gitignore? Since we move the .gitignore to top level, maybe move this too?